tools/pd-ctl/pdctl/command/scheduler_command.go

```diff
@@ -50,9 +53,47 @@ func NewSchedulerCommand() *cobra.Command {
 	c.AddCommand(NewResumeSchedulerCommand())
 	c.AddCommand(NewConfigSchedulerCommand())
 	c.AddCommand(NewDescribeSchedulerCommand())
+	c.AddCommand(NewMicroServiceCommand())
 	return c
 }
 
+// NewMicroServiceCommand returns a micro-service command.
```

Comment:
Shall we put them into member command?

Reply:
There is one command(member leader). I'm afraid the command is so similar to the mcs. Or do you think what's better command to show the mcs membership?

Reply:
Then, how about adding a new subcommand in the root command?